windows-installer/setup.nsi

```diff
@@ -15,14 +15,17 @@
 ; The parent product.
 !define PRODUCT "Stackdriver"
 
+; The service the agent is running.
+!define AGENT_SERVICE "Logging"
```

Comment:
This is probably overkill. I'd just use "Logging" as a literal everywhere below, especially given that you use "logs" as a literal in the service description.